configs/tanzucommunityedition.json

```diff
@@ -8,17 +8,23 @@
   ],
   "stop_urls": [],
   "selectors": {
-    "lvl0": {
-      "selector": "",
+    "lvl0": "header.subproject-specific a.text-logo",
+    "lvl1": ".docs-content h1",
+    "lvl2": ".docs-content h2",
+    "lvl3": ".docs-content h3",
+    "lvl4": ".docs-content h4",
+    "lvl5": ".docs-content h5",
+    "text": ".docs-content p, .docs-content li",
+    "version": {
+      "selector": "#dropdownMenuButton",
       "global": true,
       "default_value": "Documentation"
```

Comment:
Just making sure, is this `default_value` wanted?

Reply:
Thanks for pointing this out! Updated to `latest`.